tests/acceptance/gcp-secrets.bats

```diff
@@ -3,38 +3,47 @@
 #load _helpers
 #
 #SKIP_TEARDOWN=true
-VAULT_ADDR='http://127.0.0.1:8200'
-VAULT_IMAGE="${VAULT_IMAGE:-hashicorp/vault:1.9.0-rc1}"
+export VAULT_ADDR='http://127.0.0.1:8200'
+export VAULT_IMAGE="${VAULT_IMAGE:-hashicorp/vault:1.9.1}"
 
-if [[ -z SERVICE_ACCOUNT_ID ]]
+if [[ -z $SERVICE_ACCOUNT_ID ]]
 then
     echo "SERVICE_ACCOUNT_ID env is not set. Exiting.."
     exit 1
 fi
 
-if [[ -z PATH_TO_CREDS ]]
+if [[ -z $GOOGLE_CLOUD_PROJECT_ID ]]
 then
-    echo "PATH_TO_CREDS env is not set. Exiting.."
+    echo "GOOGLE_CLOUD_PROJECT_ID env is not set. Exiting.."
     exit 1
 fi
 
-if [[ -z GOOGLE_PROJECT ]]
+if [[ -z $GOOGLE_CLOUD_PROJECT_NAME ]]
 then
-    echo "GOOGLE_PROJECT env is not set. Exiting.."
+    echo "GOOGLE_CLOUD_PROJECT_NAME env is not set. Exiting.."
     exit 1
 fi
 
-if [[ -z GOOGLE_APPLICATION_CREDENTIALS ]]
+if [[ -z $GOOGLE_APPLICATION_CREDENTIALS ]]
 then
     echo "GOOGLE_APPLICATION_CREDENTIALS env is not set. Exiting.."
     exit 1
 fi
 
 export SETUP_TEARDOWN_OUTFILE=/tmp/output.log
 
+cp $GOOGLE_APPLICATION_CREDENTIALS ./creds.json
```

Comment:
we may want to also clean up this file after the tests. If we moved the `cp` to the `setup` function we should be able to safely `rm` in the teardown.

Reply:
Good point! Moved `cp` to setup and `rm` to teardown